pkgs/default.nix

```diff
@@ -216,8 +216,8 @@ pkgs: oldpkgs: {
         -Wall \
         -x c \
         "$textPath"
-    strip --strip-unneeded "$exe"
-  '';
+    ${optionalString (!pkgs.stdenv.hostPlatform.isDarwin) /* sh */ "strip --strip-unneeded \"$exe\""}
+    ''};
```

Comment:
This line should stay `'';`, i.e. it should not add a `}`. Additionally, can you use double single quotes around the call to strip? I.e.:
```
    ${optionalString (!pkgs.stdenv.hostPlatform.isDarwin) /* sh */ ''
      strip --strip-unneeded "$exe"
    ''}
```
That way we won't have to escape the embedded double quotes, and it would be more consistent with the surrounding code.

Reply:
Hm, I've just seen that the code above also uses double quotes instead of double single quotes. So scrap that, and just remove the extra `}`.

Reply:
Updated. It messes up some syntax highlighting, but works.